mne/preprocessing/ica.py

```diff
@@ -186,9 +186,7 @@ class ICA(ContainsMixin):
         are scaled to unit variance ("z-standardized") as a group by channel
         type prior to the whitening by PCA.
     %(random_state)s
-        As estimation can be non-deterministic it can be useful to fix the
-        random state to have reproducible results.
-    method : {'fastica', 'infomax', 'picard'}
+    method : 'fastica' | 'infomax' |'picard'
```

Comment:
I think what we had before was fine, see https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
I think what we had before is *consistent with the numpydoc standard* but I thought we purposefully deviated (using pipes instead of "or" or set notation)?  We use pipes throughout most of our docs I think...  If we keep @hoechenberger's suggestion, there should be a space between the last pipe and `'picard'` though

Reply:
Yes we switched to using pipe chars ~everywhere. Coming across `{}` seemed inconsistent to me while readint the documentation, esp. since other params of the same class **do** use pipes.

Reply:
I've added the missing space, thanks for the pointer, @drammock